### PR TITLE
Add test coverage for additional JMS message types

### PR DESCRIPTION
**Pull request by jamesnetherton**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

**Review by ppalaga — APPROVED** [review]

**Review by zbendhiba — APPROVED** [review]

File: integration-tests/messaging/src/main/java/org/apache/camel/quarkus/component/messaging/it/JmsRoutes.java [hunk 1]
```diff
@@ -27,11 +27,10 @@
 
     @Override
     public void configure() throws Exception {
-        // jmsMessageType text / binary routes
-        from("jms:queue:typeTest?jmsMessageType=Text")
-                .toD("jms:queue:typeTestResult?jmsMessageType=${header.type}");
+        from("jms:queue:typeTest?concurrentConsumers=5")
+                .toD("jms:queue:typeTestResult");
```
Comment (aldettinger) [inline integration-tests/messaging/src/main/java/org/apache/camel/quarkus/component/messaging/it/JmsRoutes.java:31]: Maybe, we can get rid of toD then.

**Review by aldettinger — APPROVED** [review]
A lot of good catches. Well done :+1:

File: integration-tests/messaging/src/main/java/org/apache/camel/quarkus/component/messaging/it/JmsRoutes.java (hunk 1 above)
Comment (jamesnetherton) [reply integration-tests/messaging/src/main/java/org/apache/camel/quarkus/component/messaging/it/JmsRoutes.java:31]: Ah, thanks. Thought I caught those. I'll fix.

Comment (aldettinger) [reply integration-tests/messaging/src/main/java/org/apache/camel/quarkus/component/messaging/it/JmsRoutes.java:31]: Ok, fixing in another commit is good.

**Pull request merged**